src/Functions/UniqTheta/FunctionsUniqTheta.cpp

```diff
@@ -0,0 +1,19 @@
+#include <Functions/FunctionFactory.h>
+
+#include "FunctionsUniqTheta.h"
+
+#if USE_DATASKETCHES
+
+namespace DB
+{
+
+REGISTER_FUNCTION(UniqTheta)
+{
+    factory.registerFunction<FunctionUniqThetaIntersect>();
+    factory.registerFunction<FunctionUniqThetaUnion>();
+    factory.registerFunction<FunctionUniqThetaNot>();
```

Comment:
After recent changes in https://github.com/ClickHouse/ClickHouse/pull/40711 all new functions must have an embedded documentation (see new argument `doc` in registerFunction method). Please, add documentation for these functions